kdl_parser/include/kdl_parser/kdl_parser.hpp

```diff
@@ -65,12 +65,12 @@ bool treeFromParam(const std::string& param, KDL::Tree& tree);
  */
 bool treeFromString(const std::string& xml, KDL::Tree& tree);
 
-/** Constructs a KDL tree from a TiXmlDocument
- * \param xml_doc The TiXmlDocument containting the xml description of the robot
+/** Constructs a KDL tree from a XMLDocument
+ * \param xml_doc The XMLDocument containting the xml description of the robot
  * \param tree The resulting KDL Tree
  * returns true on success, false on failure
  */
-bool treeFromXml(TiXmlDocument *xml_doc, KDL::Tree& tree);
```

Comment:
This API can't be changed in the released ROS versions since it would break API/ABI compatibility.